### PR TITLE
Fix typos in property names and comments

### PR DESCRIPTION
**Pull request by JoanGil**
## Summary

Fix typos in property names and comments:

From `propety` ❌ to `property` ✅

### CODE REVIEW

**Review by gewarren — COMMENTED** [review]
Thanks @JoanGil. I know your changes don't affect compilation, but could you add the missing C# and VB project files so we can check compilation (don't worry about the C++ one)?

**Comment by JoanGil** [comment]
What do you mean by "adding the project files"? If you could give me more context or an example I would gladly do it!

**Comment by gewarren** [comment]
> What do you mean by "adding the project files"? If you could give me more context or an example I would gladly do it!

Just adding a simple [project file like this](https://github.com/dotnet/docs/blob/main/samples/snippets/csharp/VS_Snippets_CFX/tx_enlist/cs/Project.csproj) to the following folders:

- snippets/csharp/System/Type/GetProperty
- snippets/visualbasic/VS_Snippets_CLR/Type_GetProperty1/VB

Thank you @JoanGil!

**Comment by JoanGil** [comment]
Done @gewarren!

Question: For the future, how do I know to what folders do I have to add a project file so it compiles? Thanks!

**Comment by gewarren** [comment]
> Done @gewarren!
> 
> Question: For the future, how do I know to what folders do I have to add a project file so it compiles? Thanks!

If you look at the [Files changed](https://github.com/dotnet/dotnet-api-docs/pull/9441/files) tab, you can see where there are Snippets 5000 errors. Now there are a few more since we added the projects - looks like we need to provide unique names within the project. I know this is a lot of work just for fixing a spelling mistake, so if you can't get to these, I can do it.

**Comment by JoanGil** [comment]
> If you look at the [Files changed](https://github.com/dotnet/dotnet-api-docs/pull/9441/files) tab, you can see where there are Snippets 5000 errors. Now there are a few more since we added the projects - looks like we need to provide unique names within the project. I know this is a lot of work just for fixing a spelling mistake, so if you can't get to these, I can do it.

Now I see that there are multiple errors when you compile that `csproj`. The thing is, would that be okay if I put names like: `MyClass3`, `MyTypeClass3`... It's not nice but it will solve the compiling problem. If that's okay, I can do the changes myself.

**Comment by gewarren** [comment]
> > If you look at the [Files changed](https://github.com/dotnet/dotnet-api-docs/pull/9441/files) tab, you can see where there are Snippets 5000 errors. Now there are a few more since we added the projects - looks like we need to provide unique names within the project. I know this is a lot of work just for fixing a spelling mistake, so if you can't get to these, I can do it.
> 
> Now I see that there are multiple errors when you compile that `csproj`. The thing is, would that be okay if I put names like: `MyClass3`, `MyTypeClass3`... It's not nice but it will solve the compiling problem. If that's okay, I can do the changes myself.

Yes, that's totally fine. Thanks!

**Comment by JoanGil** [comment]
Okay, I have updated the names and now the CSharp files compile. It just fails for the C++ project missing. 

Is everything done in my end, right? If not, let me know if I can do something else!

**Comment by JoanGil** [comment]
Any progress on this one? 😄

File: snippets/csharp/System/Type/GetProperty/type_getproperty2.cs [hunk 1]
```diff
@@ -18,22 +19,23 @@ public int MyProperty
         }
     }
 }
-public class MyTypeClass
+
+public class MyTypeClass22
```
Comment (gewarren) [inline snippets/csharp/System/Type/GetProperty/type_getproperty2.cs:23]:
```suggestion
public class MyTypeClass2
```

**Review by gewarren — APPROVED** [review]

**Pull request merged**